editor.planx.uk/src/ui/shared/CharacterCounter.tsx

```diff
@@ -1,32 +1,16 @@
 import Typography from "@mui/material/Typography";
 import { visuallyHidden } from "@mui/utils";
-import { TEXT_LIMITS, TextInputType } from "@planx/components/TextInput/model";
 import { debounce } from "lodash";
 import React, { useCallback, useEffect, useState } from "react";
 import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
 export type Props = {
-  textInputType: TextInputType;
+  limit: number;
```

Comment:
Retrospectively feels like a much better, more flexible approach to just pass in a `limit` here rather than type :heavy_plus_sign: